componentize.py

```diff
@@ -378,19 +389,20 @@ def create_ghuser_component(source, target, version=None, prefix=None):
         os.mkdir(targetdir)
     print("[x]")
 
-    print("[ ] GH_IO assembly: {}\r".format(gh_io or args.ghio), end="")
     if not gh_io:
-        print("[-]")
-        print("    Cannot find GH_IO Assembly! Aborting.")
+        print("[-]  Cannot find GH_IO Assembly! Aborting.")
         sys.exit(-1)
-    clr.AddReferenceToFileAndPath(gh_io)
-    print("[x]")
-    print()
+
+    gh_io = os.path.abspath(gh_io)
+    gh_io = gh_io[:-4]
```

Comment:
It chops off the extension `.dll` from the path because the new `clr module` `s `clr.AddReference()` doesn't accept it.

Maybe it can be resumed as one line like this (?):
```py
clr.AddReference(os.path.splitext(gh_io)[0])
```